__tests__/api.test.js

```diff
@@ -42,7 +42,73 @@ describe("GET /api/topics", () => {
 }),
 
 describe('GET api/articles', () => {
+  describe('successful connection tests', () => {
+    test('200: articles page returns an array containing objects of all articles', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+          expect(Array.isArray(body.articles)).toBe(true)
+        })
+    })
+    test('200: all articles should have their given properties as listed below', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({
+          body
+        }) => {
+          expect(body.articles.length).toBeGreaterThan(0)
+          for (let i = 0; i < body.articles.length; i++) {
+            expect(body.articles[i]).toHaveProperty("article_id", expect.any(Number))
+            expect(body.articles[i]).toHaveProperty("title", expect.any(String))
+            expect(body.articles[i]).toHaveProperty("author", expect.any(String))
+            expect(body.articles[i]).toHaveProperty("topic", expect.any(String))
+            expect(body.articles[i]).toHaveProperty("created_at", expect.any(String))
+            expect(body.articles[i]).toHaveProperty("votes", expect.any(Number))
+            expect(body.articles[i]).toHaveProperty("article_img_url", expect.any(String))
+          }
+        })
+    })
+    test('200: articles should be ordered by created_at in descending order', () => {
```

Comment:
Nice work with this, if you wanted to you could look into the npm package jest sorted to make asserting for the order of the response a little easier and more readable:
https://www.npmjs.com/package/jest-sorted